DataONEorg/speed-bagit
# Restructure the project to have the correct org.dataone.xyz structure

This pull request is linked to issue #7 . Its purpose is to add the additional `speedbagit` folder under `org.dataone`.

To test:
1. Clone the repository
2. See that the folder structure is correct and follows DataONE naming conventions
3. Confirm that the unit tests run without import errors

Test Failures:
Unrelated to this change; it's failing on theJava 8 build (passing on Java 11) because there isn't a toolchain configured.
